docs/docs/cmd/spo/listitem/listitem-get.md

```diff
@@ -59,3 +62,184 @@ Get an item with specific properties from a list retrieved by server-relative UR
 ```sh
 m365 spo listitem get --listUrl /sites/project-x/documents --id 147 --webUrl https://contoso.sharepoint.com/sites/project-x --properties "Title,Created,Company/Title"
 ```
+
+Get an item with ID _147_ from list with title _Demo List_ in site _https://contoso.sharepoint.com/sites/project-x_ with permissions
```

Comment:
same here. lets refactor to more general description of examples which is our current approach we should use 👍